cmd/config.yaml

```diff
@@ -0,0 +1,5 @@
+logging:
+  level: debug  # valid levels: error, warning, info, debug
+ratsd:
+  listen-addr: 0.0.0.0:8888
+  protocol: http
```

Comment:
Since we have the certs, I guess this can be https, no?

Reply:
Yes, I've updated it to `https` in the patch that brings in the required certificates.